File: bro/src/main/java/me/xx2bab/bro/core/base/IBroMonitor.java
package me.xx2bab.bro.core.base;

import me.xx2bab.bro.core.activity.Builder;

public interface IBroMonitor {

    void onActivityRudderException(int errorCode, Builder builder);

    void onModuleException(int errorCode);

    void onApiException(int errorCode);

}

File: bro/src/main/java/me/xx2bab/bro/core/base/IBroMonitor.kt
package me.xx2bab.bro.core.base

import me.xx2bab.bro.core.activity.Builder

interface IBroMonitor {
    fun onActivityRudderException(errorCode: Int, builder: Builder)
    fun onModuleException(errorCode: Int)
    fun onApiException(errorCode: Int)
}